mesheryctl/pkg/utils/helpers.go

```diff
@@ -70,6 +70,8 @@ var (
 	// DockerComposeFile is the default location within the MesheryFolder
 	// where the docker compose file is located.
 	DockerComposeFile = "meshery.yaml"
+	// ManifestsFolder is where the Kubernetes manifests are stored
+	ManifestsFolder = "manifests"
```

Comment:
There should be a `manifests` folder inside the `.meshery` folder that stores the manifest files.